uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/APIWrapper/DefaultLimitRemoverTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+
+/**
+ * Class CRM_HRCore_APIWrapper_DefaultLimitRemoverTest
+ *
+ * @group headless
+ */
+class CRM_HRCore_APIWrapper_DefaultLimitRemoverTest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, TransactionalInterface {
+
+  private $defaultLimitRemoverObj;
+
+  private $testParameters = [];
+
+  public function setUpHeadless() {
+    return \Civi\Test::headless()
+      ->installMe(__DIR__)
+      ->apply();
+  }
+
+  public function setUp() {
+    $this->defaultLimitRemoverObj = new CRM_HRCore_APIWrapper_DefaultLimitRemover();
+
+    $this->testParameters['params'] = [
+      'action' => 'get',
+      'entity' => 'contact',
+      'id' => 1,
+    ];
+  }
+
+  public function testFromApiInputWithLimitSetInParameters() {
```

Comment:
As a best practice, it's best that the test name describes how the class should behave. So, instead of `testFromApiInputWithLimitSetInParameters` a better name would be `testTheDefaultLimitShouldNotBeAppliedIfASpecificLimitHasBeenSet`

The same applies to the other test names